service/controller/appcatalog/resource/appcatalogentry/types.go

```diff
@@ -18,5 +21,8 @@ type entry struct {
 }
 
 type metadata struct {
-	Restrictions v1alpha1.AppCatalogEntrySpecRestrictions `json:"restrictions"`
+	Annotations     map[string]string                         `json:"annotations"`
+	ChartAPIVersion string                                    `json:"chartApiVersion"`
```

Comment:
```suggestion
	ChartAPIVersion string                                    `json:"chartAPIVersion"`
```

These abbreviations should be consistent and API is the correct casing IMO. But this is the casing in the metadata file isn't it?

If so lets go with this but I'll do a PR to the spec to fix the casing. The metadata files are still new so I'd like to get this right.

Reply:
Another idea. Should the struct be appMetadata instead of metadata? 

No need to export it but I got a bit confused earlier even though I think I chose the name. 😄  #NamingIsHard

Reply:
> But this is the casing in the metadata file isn't it? 
This is from metadata actually, so yes, we should fix on spec as you said. Thanks for noticing!

> Should the struct be appMetadata instead of metadata?

You're right! Since we have `metadata` from upstream as well, it's better to change the name.

Reply:
Cool I'll do a spec PR. Thanks for renaming!